xbmc/interfaces/legacy/Player.cpp

```diff
@@ -426,26 +426,26 @@ namespace XBMCAddon
       }
     }
 
-    std::vector<String>* Player::getAvailableSubtitleStreams()
+    std::vector<String> Player::getAvailableSubtitleStreams()
```

Comment:
Just changed it to guarantee it doesn't return null pointer. Does it make any difference if it returns value or pointer?